apps/block_scout_web/assets/js/lib/address_total_history_chart.js

```diff
@@ -70,7 +67,7 @@ class AddressTotalHistoryChart {
       pointRadius: 0,
       backgroundColor: sassVariables.primary,
       borderColor: sassVariables.primary,
-      lineTension: 0
+      lineTension: 0.1
```

Comment:
Not sure but maybe we could increase the value since the graph lines are still far from smooth.